src/js/apps/admin/list/clinicians-all_app.js

```diff
@@ -1,18 +1,61 @@
 import Radio from 'backbone.radio';
 
-import App from 'js/base/app';
-import { ListView, LayoutView } from 'js/views/admin/list/clinicians-all_views';
+import SubRouterApp from 'js/base/subrouterapp';
 
-export default App.extend({
+import intl from 'js/i18n';
+
+import { AddClinicianButtonView, ListView, LayoutView } from 'js/views/admin/list/clinicians-all_views';
+
+export default SubRouterApp.extend({
+  routerAppName: 'CliniciansApp',
+  eventRoutes: {
+    'clinician': 'showClinicianSidebar',
+    'clinician:new': 'showClinicianSidebar',
+  },
+  viewEvents: {
+    'click:addClinician': 'onClickAddClinician',
+  },
   onBeforeStart() {
     this.showView(new LayoutView());
     this.getRegion('list').startPreloader();
+
+    this.clinicians = Radio.request('entities', 'clinicians:collection');
+
+    const addClinicianView = this.showChildView('addClinician', new AddClinicianButtonView());
```

Comment:
so this'll be real annoying since I suggested breaking this out.. _but_ now it could be back in the layout safely... thoughts?

Reply:
Good call. I'm all for moving it back to layout.